src/Controller/ContactController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace App\Controller;
+
+use App\Form\ContactType;
+use App\Model\Contact;
+use App\Notification\ContactNotification;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait;
+
+class ContactController extends AbstractController
+{
+    /**
+     * Contact page display
+     * @Route("/contact",name="contact_index",  methods={"GET","POST"})
+     * @return Response A reponse instance
+     */
+    public function contact(Request $request, MailerInterface $mailer): Response
+    {
+        $form = $this->createForm(ContactType::class);
```

Comment:
You probably should add the contact object in the createForm instead of using the getData.
I doubt the validation is done properly in this case